UmpleToJava/UmpleTLTemplates/class_MethodDeclaration.ump

```diff
@@ -8,6 +8,8 @@ class UmpleToJava {
         
         if(!aMethod.getExistsInLanguage("Java"))
           continue;
+        if(aMethod.existsAlreadyInClass(uClass))
```

Comment:
Yes, the issue is with the methods that are generated by the UmpleTL, however, isn't there information about those methods on the model and its associations?

I was tracing through the debugger using the example snippet of code in the issue #834, and I was able to find the method on the class which was already generated, so this conditional so stop the code under this line from generating the method stub for the interface in addition to that method that was already previously generated.